.gitignore

```diff
@@ -396,3 +396,9 @@ FodyWeavers.xsd
 
 # JetBrains Rider
 *.sln.iml
+/.idea/.idea.SIT.Manager/.idea/.gitignore
+/.idea/.idea.SIT.Manager/.idea/.name
+/.idea/.idea.SIT.Manager/.idea/avalonia.xml
+/.idea/.idea.SIT.Manager/.idea/discord.xml
+/.idea/.idea.SIT.Manager/.idea/indexLayout.xml
+/.idea/.idea.SIT.Manager/.idea/vcs.xml
```

Comment:
Might want to just exclude the whole ```/.idea``` folder as there might be a whole load more files it generates in the future?

Reply:
Oh, thanks rider for generating the worst possible gitignore changes. Yeah I'll change that :3

Reply:
Hehe I wasn't sure if they were auto generated or not 🤷